### ISSUE TITLE
Investigate/fix bower legacy mode

### ISSUE THREAD

**Issue by bogdanudrescu** [vaadin/flow#5640]
As per https://vaadin.com/forum/thread/17652644/upgrading-to-14-0-0-alpha3-breaks-project, bower legacy mode seems not to work.

**Comment by caalador** [comment]
It's not required to have, so v14 will probably not recognize the project as a bower project.
Should probably work fine in npm mode after all components have been migrated and added to the platform.

**Comment by bogdanudrescu** [comment]
https://vaadin.com/start/latest/java-ui fails on 14.0.0.alpha3 only when `jetty:run` from ui module, not from root pom.

```
Caused by: java.lang.IllegalStateException: Failed to resolve the following module imports neither in the node_modules directory '/Users/bogdan/work/workspace-vaadin/bower-legacy-project/bower-legacy-project-ui/node_modules' nor in project files in '/Users/bogdan/work/workspace-vaadin/bower-legacy-project/bower-legacy-project-ui/frontend': 
'@vaadin/flow-frontend/selectConnector.js'
'vaadin-grid-flow-selection-column.js'
'@vaadin/flow-frontend/gridConnector.js'
'@vaadin/flow-frontend/contextMenuConnector.js'
'@vaadin/flow-frontend/flow-component-renderer.js'
'css/shared-styles.js'
'@vaadin/flow-frontend/dndConnector.js'
```

User's project:
```
Caused by: java.lang.IllegalStateException: Failed to resolve the following module imports neither in the node_modules directory '/Users/bogdan/work/workspace-vaadin/my-starter-project/node_modules' nor in project files in '/Users/bogdan/work/workspace-vaadin/my-starter-project/frontend': 
'@vaadin/flow-frontend/dndConnector.js'
```

This is from the fact that `validate` goal which is not executed contains some copy files in it.

`./src` is the reason for this checked in `DevModeInitializer` ```if (!new File(FrontendUtils.getBaseDir(), "src").isDirectory()```, since bookstore started has `src` only in modules, but not in root.

**Comment by manolo** [comment]
Fixed in #5650
